SECURITY_RELEASE_PROCESS.md

```diff
@@ -188,20 +188,28 @@ could be in the form of the following:
 To be eligible for the cncf-envoy-distributors-announce mailing list, your
 distribution should:
 
-1. Be an actively maintained distribution of Envoy components OR offer Envoy as a publicly
-   available service in which the product clearly states that it is built on top of Envoy. E.g.,
-   "SuperAwesomeLinuxDistro" which offers Envoy pre-built packages OR
-   "SuperAwesomeCloudProvider's Envoy as a Service (EaaS)". A cloud service that uses Envoy for a
-   product but does not publicly say they are using Envoy does not qualify.
+1. Be either:
+   1. An actively maintained distribution of Envoy components. An example is
+   "SuperAwesomeLinuxDistro" which offers Envoy pre-built packages. Another example is
+   "SuperAwesomeServiceMesh" which offers a service mesh product that includes Envoy as a component.
+   2. OR offer Envoy as a publicly available infrastructure or platform service, in which the
+   product clearly states that it is built on top of Envoy. E.g., "SuperAwesomeCloudProvider's
+   Envoy as a Service (EaaS)". An infrastructure service that uses Envoy for a product but does not
+   publicly say they are using Envoy does not qualify. This is essentially IaaS or PaaS, if you use
+   Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not qualify.
 2. Have a user base not limited to your own organization.
 3. Have a publicly verifiable track record up to present day of fixing security
    issues.
 4. Not be a downstream or rebuild of another distribution.
 5. Be a participant and active contributor in the community.
 6. Accept the [Embargo Policy](#embargo-policy) that is outlined above.
 7. Be willing to [contribute back](#contributing-back) as outlined above.
-8. Have someone already on the list vouch for the person requesting membership
-   on behalf of your distribution.
+8. Have someone already on the list vouch for the person requesting membership on behalf of your distribution.
+
+Note that Envoy maintainers are members of the Envoy security team. [Members of the Envoy security
+team](OWNERS.md#envoy-security-team) and the organizations that they represents are implicitly
+included in the private distributor list. These organizations do not need to meet the above list of
+criteria with the exception of the acceptance of the embargo policy.
```

Comment:
Should we publish the list of the members of the private distributor list for transparency? k8s does this, see https://github.com/kubernetes/security/blob/master/private-distributors-list.md

Reply:
+1 if there is prior art for doing this. We should email the list and give everyone a week or so to object before making the list public. Can you also clarify somewhere that people will be added to the public list if they ask to be on it and quality? I think we can do all of this in a follow up PR.

Reply:
I've sent an e-mail to the distributor list to make sure everyone is comfortable before we go public.

Reply:
No objection here.

Reply:
+1.

Reply:
I'm going to take care of this in a distinct PR.